bedrock/mozorg/helpers/misc.py

```diff
@@ -462,8 +462,10 @@ def absolute_url(url):
 
     if url.startswith('//'):
         prefix = 'https:'
-    else:
+    elif url.startswith('/'):
```

Comment:
@pmclanahan appreciative if you could take a quick look at the updated tests here :+1: